markets/perps-market/contracts/modules/LiquidationModule.sol

```diff
@@ -36,7 +36,7 @@ contract LiquidationModule is ILiquidationModule {
             .load()
             .liquidatableAccounts;
 
-        for (uint i = 0; i < liquidatableAccounts.length(); i++) {
+        for (uint i = 1; i <= liquidatableAccounts.length(); i++) {
```

Comment:
fixed this in my PR; we actually can't use `liquidateAccounts` directly because if there's a full account liquidation, it gets removed and we get an error.  instead i move the liquidate accounts values into an array in memory which solves the issue 👍🏽

Reply:
feel free to keep this; and i'll fix merge conflict or vice versa depending on which PR goes in first

Reply:
I'll keep it otherwise tests will fail, but will fix if your PR comes first